sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -596,7 +596,7 @@
               "value": ""
             }
           ],
-          "content": "You are eligible for <b>{{data.eligibleAmount}}</b>",
+          "content": "You are eligible for <b>${{data.eligibleAmount}}</b>",
```

Comment:
just wanted to check, do we need `$` ?

Reply:
It is a dollar sign. We have $ sign present in the form as for e.g($####) so added the $ sign.

Reply:
okay got it.